presto-common/src/test/java/com/facebook/presto/common/type/TestTypeSignature.java

```diff
@@ -296,23 +296,23 @@ public void testIsCalculated()
     public void testEnumSignature()
     {
         assertEquals(
-                parseTypeSignature("test_enum(enum:varchar{\"test\" :\"\"\"\", \"hello\": \" \" , \"a\":\"}{{\" })"),
+                parseTypeSignature("test_enum(enum:varchar{\"test\" :\"EI======\", \"hello\": \"EA======\" , \"a\":\"PV5XW===\" })"),
```

Comment:
Why are only the values that are encoded with base32? Are the keys not prone to this problem?

Reply:
the keys must be case-insensitive because they can be used as qualified names (`SELECT country.UK`) which must be case-insensitive according to the SQL spec iiuc